### PR TITLE
Quickly mute by double-clicking the sound button

### PR DESCRIPTION
**Pull request by urkerab**
Just for convenience.

### CODE REVIEW

File: js/client-topbar.js [hunk 1]
```diff
@@ -44,7 +44,12 @@
 			app.addPopup(LoginPopup);
 		},
 		openSounds: function () {
-			app.addPopup(SoundsPopup);
+			if (!app.addPopup(SoundsPopup)) {
```
Comment (Zarel) [inline js/client-topbar.js]: This seems to happen whenever the sound button is closed, rather than only on double-click?

Comment (urkerab) [reply js/client-topbar.js]: I can measure the time interval between the two clicks if you care to tell me what interval you think is appropriate...

Comment (Zarel) [reply js/client-topbar.js]: You can also just sidestep manually time-intervaling by listening to the `doubleclick` event.

Comment (urkerab) [reply js/client-topbar.js]: I found out where I originally went wrong with that approach - it's called `dblclick`.

File: js/client-topbar.js [hunk 2]
```diff
@@ -5,6 +5,7 @@
 			'click a': 'click',
 			'click .username': 'clickUsername',
 			'click button': 'dispatchClickButton',
+			'dblclick .volume': 'toggleMute',
```
Comment (Zarel) [inline js/client-topbar.js]: `dblclick button[name=openSounds]`

You don't need the `.volume` class.

Comment (urkerab) [reply js/client-topbar.js]: Sorry, I'm unfamiliar with Backbone's View system (I had to look it up just to find out what it was doing) so I resorted to blithely cutting and pasting...

**Pull request merged**